doc/user_guide/mva.rst

```diff
@@ -166,11 +156,80 @@ To perform Poissonian noise normalization:
      The long way:
      >>> s.decomposition(normalize_poissonian_noise=True)
 
-     Because it is the first argument we cold have simply written:
+     Because it is the first argument we could have simply written:
      >>> s.decomposition(True)
 
 More details about the scaling procedure can be found in [Keenan2004]_.
 
+Robust principal component analysis
+-----------------------------------
+
+PCA is known to be very sensitive to the presence of outliers in data. These outliers
+can be the result of missing or dead pixels, X-ray spikes, or very low count data.
+If one assumes a dataset to consist of a low-rank component **L** corrupted by
+a sparse error component **S**, then Robust PCA (RPCA) can be used to recover the
+low-rank component for subsequent processing.
+
+The default RPCA algorithm returns the factors and loadings of **L**, and can be
+accessed with the following code. You must set the "output_dimension" when using
+RPCA.
```

Comment:
Could you add the reference for the RPCA algorithm? Also, as at sklearn they're implementing Candes' one, wouldn't it be better to call this one RPCA_<algorith surname> to differentiate them in case we decide to wrap sklearn's too?

And, given that rpca is an acronym, would't "RPCA" and "ORPCA" be better?